rasa/shared/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class RasaException(Exception):
+    """Base exception class for all errors raised by Rasa."""
```

Comment:
(definitely optional): I wonder if we really need a dedicated file for exceptions since there is only one very very generic one.

Reply:
I thought more would come soon 🤔  I agree with you that we should have a look on the final package and then revisit what we actually still need or don't.